### ISSUE TITLE
Can't locate path: </home/reka/Desktop/tcharter/vendor/tje3d/telegram/src/Laravel/config.php>

### ISSUE THREAD

**Issue by mrmrn** [tje3d/Telegram#3]
Hi,
I tried to install this new version on laravel 5.4.
after adding facade and aliase when I try php artisan vendor:publish --tag=telegram it gave me this error:
```
Can't locate path: </home/reka/Desktop/tcharter/vendor/tje3d/telegram/src/Laravel/config.php>
Publishing complete
```

and config/telegram.php does not exist in the project.

**Comment by mrmrn** [comment]
you used vendor/tje3d/telegram/src/Laravel/Config.php and I renamed it to config.php and it works now.
```
reka@marn ~/D/tcharter> php artisan vendor:publish --tag=telegram
Copied File [/vendor/tje3d/telegram/src/Laravel/config.php] To [/config/telegram.php]
Publishing complete.
```

**Comment by tje3d** [comment]
Thanks for reporting, fixed in 39aba8a39a9a31c1e00f5a912b6d0add2e63b858
